src/Testing/Resources/CachedContentsTestResource.php

```
@@ -0,0 +1,57 @@
+
+final class CachedContentsTestResource implements CachedContentsContract
+{
+    use Testable;
+
+    protected function resource(): string
+    {
+        return \Gemini\Resources\CachedContents::class;
+    }
+
+    public function create(
+        BackedEnum|string $model,
```

PHPMD (2.15.0)

Avoid unused parameters such as '$model'. (Unused Code Rules)

(UnusedFormalParameter)